### PR TITLE
fix: add verbosity flags to resume command

### PR DESCRIPTION
**Pull request by EstebanRomero84**
### What does this PR do?
Adds verbosity flags to resume command.
### What issues does this PR fix or reference?

[Insert GUS WI](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001NG4OJYA1/view)

### Functionality Before

**sf project deploy pipeline resume** does not have --concise and --verbose flags.

### Functionality After

**sf project deploy pipeline resume** has --concise and --verbose flags.

### CODE REVIEW

File: test/commands/project/deploy/pipeline/resume.test.ts [hunk 1]
```diff
@@ -177,6 +188,28 @@ describe('project deploy pipeline resume', () => {
...
+          sf_devops__Status__c: AsyncOperationStatus.InProgress,
+          sf_devops__Error_Details__c: 'mockErrorDetail',
+        };
+        sandbox.stub(AorSelector, 'selectAsyncOperationResultById').resolves(mockAorRecord);
+
+        monitorStub = sinon.stub(AsyncOpStreaming.prototype, 'monitor');
+        sandbox.stub(ResumeCommandOutputService.prototype, 'getStatus').returns(AsyncOperationStatus.Completed);
+        stubDisplayEndResults = sandbox.stub(ResumeCommandOutputService.prototype, 'displayEndResults');
+      })
+      .command(['project deploy pipeline resume', `-i=${mockAorId}`, '--verbose'])
```
Comment (JuanStenghele-sf) [inline test/commands/project/deploy/pipeline/resume.test.ts:207]: How about a test to check that stubDisplayEndResults is not called when the verbose flag is not used? And another one for the concise case (the output should be the same as if it was not used)?

**Review by JuanStenghele-sf — CHANGES_REQUESTED** [review]
This looks really good! Just some test suggestions and we should also update the readme.md with the concise and verbose flags info.

File: test/commands/project/deploy/pipeline/resume.test.ts (hunk 1 above)
Comment (EstebanRomero84) [reply test/commands/project/deploy/pipeline/resume.test.ts:207]: Good catch. I updated the tests and the READE file.
Thanks.

**Review by JuanStenghele-sf — APPROVED** [review]
Great job!

**Pull request merged**